torch_xla/csrc/tensor.cpp

```diff
@@ -1372,6 +1372,9 @@ XLATensor XLATensor::mean(const XLATensor& input,
                           std::vector<xla::int64> dimensions,
                           bool keep_reduced_dimensions,
                           c10::optional<at::ScalarType> dtype) {
+  for (auto& d : dimensions) {
```

Comment:
We probably need an API in helpers.* in order to avoid open-coding loops like this all over the places.
Also GetCanonicalDimension() should go to helpers.*